OpenRA.Mods.Common/Traits/CombatDebugOverlay.cs

```diff
@@ -28,14 +30,14 @@ public class CombatDebugOverlay : IPostRender, INotifyDamage
 		readonly DeveloperMode devMode;
 
 		readonly HealthInfo healthInfo;
-		readonly BlocksProjectilesInfo blockInfo;
+		readonly IBlocksProjectiles[] blocker;
 		Lazy<AttackBase> attack;
 		Lazy<BodyOrientation> coords;
 
 		public CombatDebugOverlay(Actor self)
 		{
 			healthInfo = self.Info.TraitInfoOrDefault<HealthInfo>();
-			blockInfo = self.Info.TraitInfoOrDefault<BlocksProjectilesInfo>();
+			blocker = self.TraitsImplementing<IBlocksProjectiles>().ToArray();
```

Comment:
Any traits implementing `IBlocksProjectiles` are not guaranteed to have been created by the time you perform this lookup. Either wrap it in a `Lazy<T>` like the others do, or implement `INotifyCreated` and do your trait lookups in there (the traits will be ready by then).